README.md

```diff
@@ -1,24 +1,27 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+#Programming Learning Journal 
+ This is a Ruby on Rails 5 application that will help you keep track of your progress as you gain experience with a variety of languages, libraries, frameworks and technologies.
+
+##Getting Started
+To get the Rails server running locally:
+
+*Clone this repo
+*bundle install to install all req'd dependencies
+*rake db:migrate to make all database migrations
```

Comment:
`rails db:create db:migrate` instead of `rake db:migrate`, on first run we need to create the database first!

It's a rake task but they've moved it into rails namespace now as well